docs/tutorials/rootless_tutorial.md

```diff
@@ -22,6 +22,8 @@ The [slirp4netns](https://github.com/rootless-containers/slirp4netns) package pr
 
 When using Podman in a rootless environment, it is recommended to use fuse-overlayfs rather than the VFS file system.  Installing the fuse3-devel package gives Podman the dependencies it needs to install, build and use fuse-overlayfs in a rootless environment for you.  The fuse-overlayfs project is also available from [GitHub](https://github.com/containers/fuse-overlayfs).  This especially needs to be checked on Ubuntu distributions as fuse-overlayfs is not generally installed by default.
 
+After installing fuse-overlayfs, in `$HOME/.config/containers/storage.conf` change `driver` under `[storage]` to `"overlay"`.
```

Comment:
This should be done automatically if you've never used Podman before, so this should probably say it "may be necessary to change the `driver` option..." or similar